content/05_labs/01_sso/_index.md

```diff
@@ -1,20 +1,20 @@
 ---
-title: "Single Sign-On"
+title: "Signing in securely"
```

Comment:
What about "Secured Access" or "Secure Access to AWS" or something like that?

Reply:
I went with "Secure Access to AWS"